code/game/objects/items/weapons/airlock_painter.dm

```diff
@@ -33,14 +33,52 @@
 //because you're expecting user input.
 /obj/item/weapon/airlock_painter/proc/can_use(mob/user)
 	if(!ink)
-		user << "<span class='notice'>There is no toner cardridge installed installed in \the [name]!</span>"
+		user << "<span class='notice'>There is no toner cartridge installed installed in \the [name]!</span>"
 		return 0
 	else if(ink.charges < 1)
 		user << "<span class='notice'>\The [name] is out of ink!</span>"
 		return 0
 	else
 		return 1
 
+/obj/item/weapon/airlock_painter/suicide_act(mob/user)
+	if(can_use(user))
+		user.visible_message("<span class='suicide'>[user] is inhaling toner from \the [name]! It looks like \he's trying to commit suicide.</span>")
+		use(user)
+
+		// Wait two beats for the toner to take effect
+		sleep(20)

```

Comment:
now you have to put a bunch of sanity checks after this to make sure the user still exists